### DISCUSSION TITLE
Xavier的DOTA2RPG讨论区

### DISCUSSION THREAD

**Discussion by XavierCHN** [XavierCHN/x-template#26] [Announcements]
## 👋 欢迎来到DOTA2RPG讨论区
 在这个讨论区，你可以
  *  问关于DOTA2RPG制作的任何问题
  * 分享你关于DOTA2RPG制作的一些发现
等等

**Comment by aaaazdxzds** [comment]
X大，我想学TS做前端，但是发现和JS区别还挺大的，请问有啥项目可以参考吗

**Reply by XavierCHN** [reply]
没有太大本质区别，更多的还是纯粹的前端技术，所以如果用react开发的话，去学习相应的教程就可以了

**Reply by aaaazdxzds** [reply]
好的，谢谢X大

**Comment by LukeGongde** [comment]
X大，有详细一些的教程吗，新手
